tasks/models.py

```diff
@@ -30,8 +32,10 @@ def __str__(self) -> str:
 
 
 class Comment(models.Model):
-    user_id = models.ForeignKey(User, on_delete=models.RESTRICT, related_name='comments')
-    task_id = models.ForeignKey(Task, on_delete=models.CASCADE)
+    user = models.ForeignKey(
+        User, on_delete=models.RESTRICT, related_name='comments')
```

Comment:
maybe
```suggestion
        User, on_delete=models.RESTRICT, related_name='user')
```

Reply:
why?

Reply:
it should stay  "comments" im my opinion , cause when I will need to pull all the comments of a user I will write user.comments, and not user.user